.travis.yml

```diff
@@ -3,16 +3,19 @@ language: go
 sudo: false
 
 go:
-  - "1.10.2"
+  - "1.11.x"
+
+env:
+  - GO111MODULE=on
 
 install:
-  - go get golang.org/x/tools/cmd/cover
-  - go get github.com/mattn/goveralls
-  - go get github.com/kisielk/errcheck
-  - go get ./...
+  # Install `cover` and `goveralls` without `GO111MODULE` enabled so that we
+  # don't download project dependencies and just put the tools in our gobin.
```

Comment:
> I think go get with GO111MODULE=off still fetches dependencies, since that's what go get has always done, right? 

It downloads the dependencies needed by these tools, into the default `$GOPATH` (`$HOME/go`) and that's what I wanted. 

I didn't make it very clear in my comment but with `GO111MODULE=on` running `go get github.com/cpu/some-tool-we-need` in the `ct-woodpecker` project root with Go 1.11+  would download all of the dependencies for `ct-woodpecker` **and** `some-tool-we-need`. I don't really understand why that is the case but what I wanted was to just let a default `GOPATH` be initialized with the various utils we need and their deps, without fetching `ct-woodpecker`'s deps. I also didn't want to vendor the tools in `ct-woodpecker`.

Does that make sense? I could try to better capture this in the comment.

Reply:
Ohhh I misread "project dependencies" to mean the opposite - the dependecies of `cover` and `goveralls`. I think if you just replace "project" with "ct-woodpecker" it becomes clear (though I also don't understand why Go has the behavior you described)

Reply:
> I think if you just replace "project" with "ct-woodpecker" it becomes clear 

55860dc

> I also don't understand why Go has the behavior you described

Thinking about it more it happens because running `go get` in a module enabled project with `GO111MODULE=on` means you want `go get` to be updating `go.mod` to track the new dependency. I think at the same time it adds the new dependency it fetches the others. I thought about trying to run the `go get` outside of the project dir instead but while researching the answer to this I see that would have caused its own unique problem specifically because `go get` would have failed to find the `go.mod` to update: https://github.com/golang/go/wiki/Modules#why-does-installing-a-tool-via-go-get-fail-with-error-cannot-find-main-module